index.html

```diff
@@ -0,0 +1,333 @@
+
+        <section class="home-why-us">
+            <div class="home-why-us__content">
+                <p class="home-why-us__label">Why should you why-work-together with us?</p>
+                <h2 class="home-why-us__title">To upscale your business <br> to the next level</h2>
+                <p class="home-why-us__description">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy <br>
+                    eirmod tempor invidunt
+                    ut labore et dolore magna aliquyam erat, sed diam <br> voluptua invidunt ut labore.</p>
+            </div>
+
+            <div class="home-why-us__list">
+                <div class="home-why-us__list-item">
+                    <img class="home-why-us__list-icon" src="/img/Icon.svg" alt="">
+                    <p class="home-why-us__list-description">Lorem ipsum dolor sit amet,
+                        consectetur adipiscing elit, <br> sed do eiusmod.</p>
+                </div>
+                <div class="home-why-us__list-item">
+                    <img class="home-why-us__list-icon" src="/img/Icon.svg" alt="">
+                    <p class="home-why-us__list-description">Lorem ipsum dolor sit amet,
+                        consectetur adipiscing elit, <br> sed do eiusmod.</p>
+                </div>
+                <div class="home-why-us__list-item">
+                    <img class="home-why-us__list-icon" src="/img/Icon.svg" alt="">
+                    <p class="home-why-us__list-description">Lorem ipsum dolor sit amet,
+                        consectetur adipiscing elit, <br> sed do eiusmod.</p>
+                </div>
+            </div>
+        </section>
+
+        <section class="home__conversions">
+            <div class="home__conversions-content">
+                <div class="home__conversions-content-text">
+                    <h2 class="home__conversions-content-title">More impressions, more conversions</h2>
+                    <p class="home__conversions-content-paragraph">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy <br>
+                        eirmod tempor
+                        invidunt ut labore et dolore magna aliquyam erat, sed diam <br> voluptua invidunt ut labore.</p>
+                    <button
+                        class="home__conversions-content-button">Get
+                        Free trial</button>
+                </div>
+                <div>
+                    <img class="home__conversions-content-image" src="/img/Image.png" alt="">
+                </div>
+            </div>
+        </section>
+
+        <section class="home__different-logos">
+            <img src="/img/Logo 1.svg" class="home__logo" alt="">
+            <img src="/img/Logo 2.svg" class="home__logo" alt="">
+            <img src="/img/Logo 3.svg" class="home__logo" alt="">
+            <img src="/img/Logo 4.svg" class="home__logo" alt="">
+            <img src="/img/Logo 5.svg" class="home__logo" alt="">
+        </section>
```

Comment:
## name wrong
change in this two sections because you are using element outside its block